### PR TITLE
Added pay by card icon

### PR DESCRIPTION
**Pull request by andrii-onufriichuk**

### CODE REVIEW

File: view/frontend/web/css/source/_module.less [hunk 1]
```diff
@@ -68,6 +68,10 @@
                     margin-left: 5px;
                 }
 
+                .rvvup-card-img {
+                    width: auto !important;
```
Comment (tomaszrvvup) [inline view/frontend/web/css/source/_module.less:72]: **❔ question:** Why is this required for cards specifically?

**Review by tomaszrvvup — APPROVED** [review]

File: view/frontend/web/css/source/_module.less (hunk 1 above)
Comment (andrii-onufriichuk) [reply view/frontend/web/css/source/_module.less:72]: Otherwise, it won't be displayed for a full size.
We have other specific one for rvvup-yapily-img

Comment (aashwin-rvvup) [reply view/frontend/web/css/source/_module.less:72]: Tested this out to see how it looks:
image

**Review by jay-rvvup — APPROVED** [review]

**Review by aashwin-rvvup — APPROVED** [review]
Could we merge this one too?
